src/Microsoft.ML.PCA/PcaTrainer.cs

```diff
@@ -33,12 +33,35 @@ namespace Microsoft.ML.Trainers
     // REVIEW: make RFF transformer an option here.
 
     /// <summary>
-    /// This trainer trains an approximate PCA using Randomized SVD algorithm
-    /// Reference: https://web.stanford.edu/group/mmds/slides2010/Martinsson.pdf
+    /// The <see cref="IEstimator{TTransformer}"/> to create a model that computes anomaly scores on unlabeled examples.
     /// </summary>
     /// <remarks>
-    /// This PCA can be made into Kernel PCA by using Random Fourier Features transform
+    /// <format type="text/markdown"><![CDATA[
+    /// To create this trainer, use [RandomizedPca](xref:Microsoft.ML.PcaCatalog.RandomizedPca(Microsoft.ML.AnomalyDetectionCatalog.AnomalyDetectionTrainers,System.String,System.String,System.Int32,System.Int32,System.Boolean,System.Nullable{System.Int32}))
+    /// or [RandomizedPca(Options)](xref:Microsoft.ML.PcaCatalog.RandomizedPca(Microsoft.ML.AnomalyDetectionCatalog.AnomalyDetectionTrainers,Microsoft.ML.Trainers.RandomizedPcaTrainer.Options)).
+    ///
+    /// [!include[io](~/../docs/samples/docs/api-reference/io-columns-anomaly-detection.md)]
+    ///
+    /// ### Trainer Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Machine learning task | Anomaly Detection |
+    /// | Is normalization required? | Yes |
+    /// | Is caching required? | No |
+    /// | Required NuGet in addition to Microsoft.ML | None |
+    ///
+    /// ### Training Algorithm Details
+    /// This trainer trains an approximate PCA using a randomized method for computing the singular value decomposition (SVD) of
+    /// the matrix whose rows are the input vectors.
+    /// The method is described [here](https://web.stanford.edu/group/mmds/slides2010/Martinsson.pdf).
```

Comment:
My wish is that user can see the scoring rule in this document.

Reply:
I added it, but are you sure this is needed? isn't it too much math?